== TEST (2019-03-31 13:47:32) 
-- CODE:
n1=1
n2=2
n3=3
n4=4
print(n1_n2_n3_n4)

-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 5, in <module>
    print(n1_n2_n3_n4)
NameError: name 'n1_n2_n3_n4' is not defined
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-31 13:49:11) 
-- CODE:
n1=1
n2=2
n3=3
n4=4
print(n1)(n2)(n3)(n4)

-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 5, in <module>
    print(n1)(n2)(n3)(n4)
TypeError: 'NoneType' object is not callable
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-31 13:49:54) 
-- CODE:
n1=1
n2=2
n3=3
n4=4
print(n1+n2+n3+n4)

-- OUTPUT:
10
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-03-31 13:51:14) 
-- CODE:
n1=1
n2=2
n3=3
n4=4
print(n1)print(n2)

-- ERROR:
File "XXXX", line 5
    print(n1)print(n2)
                 ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
